specification/2.0/README.md

```diff
@@ -3900,67 +3888,144 @@ Application-specific data.
 
-Simplified implementation of Fresnel from [An Inexpensive BRDF Model for Physically based Rendering](https://www.cs.virginia.edu/~jdl/bib/appearance/analytic%20models/schlick94b.pdf) by Christophe Schlick.
+```
+                                (1 - E_m(VdotN, alpha)) * (1 - E_m(LdotN, alpha))
+multiscatter_microfacet_brdf =  --------------------------------------------------
+                                            pi * (1 - E_mavg(alpha))
 
-![](figures/lightingF.PNG)
+           1                 20
+F_mavg = ---- * baseColor + ----
+          21                 21
 
-### Geometric Occlusion (G)
+                             F_mavg^2 E_mavg(alpha)
+multiscatter_fresnel = ---------------------------------
+                       1 - F_mavg * (1 - E_mavg(alpha))
+```
 
-**Smith Joint GGX**
+Although it is possible to compute the multi-scattering approximation in real-time (see [McAuley (2019): A Journey Through Implementing Multiscattering BRDFs and Area Lights](http://i3dsymposium.github.io/2019/keynotes/I3D2019_keynote_StephenMcAuley.pdf)), implementations may omit it to improve performance.
 
-[Understanding the Masking-Shadowing Function in Microfacet-Based BRDFs](http://jcgt.org/published/0003/02/03/paper.pdf) by Eric Heitz.
+### Dielectric BRDF
 
-![](figures/lightingG.PNG)
+Dielectric materials like plastics are modeled as a GGX microfacet BRDF coupled with a diffuse BRDF via the Schlick Fresnel term in the `fresnel_mix` operation. Microfacet BRDF and multi-scattering approximation are the same as in the metal BRDF. The diffuse reflection is modeled with a Lambertian BRDF.
 
-### Microfacet Distribution (D)
+```
+                 1
+diffuse_brdf = ----
+                pi
+```
+
+The Fresnel term in the mixing operation and its multi-scattering equivalent are now computed with the index of refraction, which is fixed at 1.5.
+
+```
+f0 = ((1-ior)/(1+ior))^2 = 0.04
+fresnel = f0 + (1 - f0) * (1 - HdotV)^5
+
+           1          20
+F_mavg = ---- * f0 + ----
+          21          21
+```
 
-**Trowbridge-Reitz**
+An energy-conserving and energy-preserving coupling can be achieved via the same albedo-scaling technique as in the metal BRDF, as shown in [Kulla and Conty (2017): Revisiting Physically Based Shading at Imageworks](https://blog.selfshadow.com/publications/s2017-shading-course/imageworks/s2017_pbs_imageworks_slides_v2.pdf). 
 
-Implementation of microfacet distrubtion from [Average Irregularity Representation of a Roughened Surface for Ray Reflection](https://www.osapublishing.org/josa/abstract.cfm?uri=josa-65-5-531) by T. S. Trowbridge, and K. P. Reitz
+The `fresnel_mix` operation weights the (multi-scatter) microfacet BRDF with the Fresnel term, and the diffuse BRDF  with `diffuse_weight`.
 
-![](figures/lightingD.PNG)
+```
+dielectric_brdf =
+  diffuse_brdf() * diffuse_weight(VdotN, LdotN, f0) +
+  microfacet_brdf(alpha) * fresnel(HdotV, f0) +
+  multiscatter_microfacet_brdf(alpha) * multiscatter_fresnel(f0)
+```
```

Comment:
Is this equation correct? I don't see where baseColor factors into dielectric materials.  Is this f0 (0.04) or the full reflectance color (normal incidence angle)?

Reply:
In dielectric materials, baseColor defines the color of the diffuse component. It's missing in this equation, fixed it.